app/views/feeds/_form_distribution.html.erb

```diff
@@ -61,6 +61,17 @@
         </div>
       <% end %>
 
+      <% unless feed.persisted? %>
+        <div class="col-12 mb-4">
```

Comment:
Can you change the `mb-4` to `mb-2`? Would have to do it on the prior checkbox as well. This would improve the spacing.